docs/walkthrough/working-with-the-catalog.md

```diff
@@ -39,39 +48,52 @@ Files are the content of your package.
+The metadata can be added with JSON editor, represented as a key/value table
+with infinite nesting. If workflow contains JSON schema, you will have
+predefined key/value pairs according to the schema.
 
 #### JSON editor
 
-To add a new key/value field double click on an empty cell and type key name, then press "Enter" or "Tab", or click outside of the cell. To change value double click on that value.
+To add a new key/value field double click on an empty cell and type key name,
+then press "Enter" or "Tab", or click outside of the cell. To change value
+double click on that value.
 
-Values can be strings, numbers, arrays, or objects. Every value that you type will be parsed as JSON.
+Values can be strings, numbers, arrays, or objects. Every value that you type
+will be parsed as JSON.
 
 We don't support references and compound types yet.
 
 ### Push to bucket
 
-You can push the existing package from one bucket to another. To use this feature consult [workflows](../advanced-features/workflows.md) page.
+You can push the existing package from one bucket to another. To use this
+feature consult [workflows](../advanced-features/workflows.md) page.
 
 ### Summarize
 
-Adding a `quilt_summarize.json` file to a data package (or S3 directory path) will enable content preview right on the landing page.
+Adding a `quilt_summarize.json` file to a data package (or S3 directory path)
+will enable content preview right on the landing page.
 
-![](../imgs/catalog_package_landing_page.png)
+![Package landing page](../imgs/catalog_package_landing_page.png)
 
-Colocating data with context in this way is a simple way of making your data projects approachable and accessible to collaborators.
+Colocating data with context in this way is a simple way of making your data
+projects approachable and accessible to collaborators.
 
-`quilt_summarize.json` can be a list of paths to files in S3 that you want to include in your summary. For example: `["description.md", "../notebooks/exploration.ipynb"]`. Additionally, note that if a `README.md` file is present, it will always be rendered as well.
+`quilt_summarize.json` can be a list of paths to files in S3 that you want to
+include in your summary. For example:
+`["description.md", "../notebooks/exploration.ipynb"]`. Additionally, note that
+if a `README.md` file is present, it will always be rendered as well.
 
 > There are currently some small limitations with preview:
 >
-> * Objects linked to in `quilt_summarize.json` are always previewed as of the latest version, even if you are browsing an old version of a package.
-> * Object titles and image thumbnails link to the file view, even if you are in the package view.
+> * Objects linked to in `quilt_summarize.json` are always previewed as of the
+>   latest version, even if you are browsing an old version of a package.
+> * Object titles and image thumbnails link to the file view, even if you are in
+>   the package view.
```

Comment:
Is this still true? If so, it should be moved to catalog/visualizationdashboards#quilt_summarize.json and deleted from this page.

Reply:
Yes, it's true

I moved it to Visualizations and Dashboards page